cirq/google/programs.py

```diff
@@ -68,8 +68,8 @@ def _x_to_proto_dict(gate: ops.XPowGate, q: ops.QubitId) -> Dict:
     exp_w = {
         'target': cast(devices.GridQubit, q).to_proto_dict(),
         'axis_half_turns':
-            xmon_gates.XmonGate.parameterized_value_to_proto_dict(0),
-        'half_turns': xmon_gates.XmonGate.parameterized_value_to_proto_dict(
+            xmon_gates._parameterized_value_to_proto_dict(0),
```

Comment:
always slightly strange when a module calls another modules _private method.  Maybe it doesn't belong in cirq.google but might still make it non _private.

Reply:
Yup. There's an upcoming change where the xmon gates file is disappearing and will merge into programs.py.